schema/example.go

```diff
@@ -0,0 +1,173 @@
+package schema
+
+func NewExample() Schema {
+	location := locationSchema{
+		Lat:  35.712758,
+		Long: 51.392114,
+	}
+
+	score := scoreSchema{
+		Type:  "GPA",
+		Value: "3.4",
+	}
+
+	resource := resourceSchema{
+		URL:   "http://www.example.com/my-example-slides/",
+		Label: "Slides",
+	}
+
+	core := coreSchema{
+		Name:              "John Doe",
+		Title:             "Software Engineer",
+		Image:             "example.com/Abcxyz",
+		Email:             "lucas@example.com",
+		Phone:             "http://www.example.com/my-slides/",
+		URL:               "http://www.example.com/",
+		Summary:           "The man who sold the world!",
+		CurrentLocation:   location,
+		PermanentLocation: location,
+	}
+
+	work := workSchema{
+		Name:        "XYZ Inc",
+		Description: "A social media company",
+		Position:    "Software Engineer",
+		Location:    location,
+		URL:         "http:/xyz.example.com",
+		StartDate:   "2017-12-29",
+		EndDate:     "2018-12-29",
+		Summary:     "Developing and maintaining the company website using syna",
+		Highlights:  []string{"Worked with mobile team at Twitter to develop remote debugging tools for mobile browsers"},
+	}
+
+	education := educationSchema{
+		Institution: "e.g. XYZ Institute of Technology",
```

Comment:
``` suggestion
		Institution: "XYZ Institute of Technology",
```

Reply:
Fixed in 570748d476f068d3be54b49ed54ae44398a2eeaa